src/main/java/com/arpnetworking/clusteraggregator/configuration/ClusterAggregatorConfiguration.java

```diff
@@ -621,6 +661,9 @@ public Builder setCalculateClusterAggregations(final Boolean value) {
         @NotNull
         private Duration _reaggregationTimeout = Duration.ofMinutes(1);
         @NotNull
+        @ValidateWithMethod(methodName = "validateAggregatorLivelinessTimeout", parameterType = Duration.class)
+        private Duration _aggregatorLivelinessTimeout = _reaggregationTimeout.multipliedBy(2);
```

Comment:
This line is incorrect, I just ran into a validation problem in CI. I'll address in a separate PR